index.html

```diff
@@ -0,0 +1,191 @@
+                <div id="magnifying-glass"></div>
+            </div>
+        </div>
+        <div class="right-icons">
+            <img class="icons" src="Essentials/camera-icon.svg">
+            <img class="icons" src="Essentials/squares.svg">
+            <img class="icons" src="Essentials/bell.svg">
+            <div class="icon" id="top">W</div>
+        </div>
+    </nav>
+    <div class="contents">
+        <section>
+            <div class="video">
+                <iframe width="720" height="315" src="https://www.youtube.com/embed/V74l_zS1x8E"
+                    title="YouTube video player" frameborder="0"
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+            </div>
+            <div class="video-info">
+                <h1 class="video-title">Contributing to Open Source Part I: The Easy Way</h1>
+            </div>
+            <div class="topInfo">
+                <div class="channel">
+                    <img class="channel-icon" src="Essentials/channel-icon.jpg">
+                    <div class="channel-info">
+                        <p class="channel-name">The Odin Project</p>
+                        <p class="subs">1.11K subscribers</p>
+                    </div>
+
+                    <button class="subscribe-btn">SUBSCRIBE</button>
+
+                </div>
+
+                <div class="video-icons">
+                    <a href="" class="up-btn">
+                        <img src="Essentials/thumbs-up.svg" class="thumbs-up">
+                        <span>488</span>
+                    </a>
+                    <a href="" class="down-btn">
+                        <img src="Essentials/thumbs-down.svg" class="thumbs-down">
+                        <span>Dislike</span>
+                    </a>
+                    <a href="" class="share-btn">
+                        <img src="Essentials/arrow.svg" class="share">
+                        <span>SHARE</span>
+                    </a>
+                    <a href="" class="download-btn">
+                        <img src="Essentials/download.svg" class="save">
+                    </a>
+                    <a href="" class="save-btn">
+                        <img src="Essentials/add.svg" class="save">
+                    </a>
+                </div>
+            </div>
+
+            <!-- Comments section -->
+            <div class="comment-section">
+                <div class="comment-info">
+                    <div class="comment-qty">13,448 Comments</div>
+                    <div class="sort-by">
+                        <div class="bar1"></div>
+                        <div class="bar2"></div>
+                        <div class="bar3"></div>
+                    </div>
+                    <span class="sort-text">SORT BY</span>
+                </div>
+            </div>
+        </section>
+        <!-- Right side Thumbnails -->
```

Comment:
Please try to refrain from using comments in your codebase. Comments in the code make it unclean.